platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/properties/PropertiesLoaderTest.java

```diff
@@ -0,0 +1,200 @@
+
+    Assert.assertEquals("Returns empty properties", emptyProperties, testProperties);
+  }
+
+  @Test
+  public void testLoadPropertiesWithEmptyPropertiesFile() throws Exception {
+    File emptyFile = temporaryFolder.newFile("tempFile");
+    Properties testProperties = PROPERTIES_LOADER.loadProperties(emptyFile.getPath(), null);
+    Properties emptyProperties = new Properties();
+
+    Assert.assertEquals("Returns empty properties", emptyProperties, testProperties);
+  }
+
+  @Test
+  public void testLoadPropertiesWithProperties() throws Exception {
+    Properties testProperties = PROPERTIES_LOADER.loadProperties(propertiesFile.getCanonicalPath());
+
+    Assert.assertEquals(
+        "Returns the correct properties", exampleMap.entrySet(), testProperties.entrySet());
+  }
+
+  @Test
+  public void testAttemptLoadWithSpring() throws Exception {
+    Properties testProperties =
+        PropertiesLoader.attemptLoadWithSpring(
+            PROPERTIES_FILENAME, this.getClass().getClassLoader());
+
+    Assert.assertEquals(
+        "Returns the correct properties", exampleMap.entrySet(), testProperties.entrySet());
+  }
+
+  @Test
+  public void testAttemptLoadWithSpringFileDNE() throws Exception {
+    Properties testProperties = PropertiesLoader.attemptLoadWithSpring(NON_EXISTENT_FILENAME, null);
+
+    Assert.assertEquals(
+        "Returns empty properties", Collections.emptySet(), testProperties.entrySet());
+  }
+
+  @Test
+  public void testAttemptLoadWithSpringAndClassLoader() throws Exception {
+    Properties testProperties =
+        PropertiesLoader.attemptLoadWithSpringAndClassLoader(
+            PROPERTIES_FILENAME, this.getClass().getClassLoader());
+
+    Assert.assertEquals(
+        "Returns the correct properties", exampleMap.entrySet(), testProperties.entrySet());
+  }
+
+  @Test
+  public void testAttemptLoadWithSpringAndClassLoaderFileDNE() throws Exception {
+    Properties testProperties =
+        PropertiesLoader.attemptLoadWithSpringAndClassLoader(NON_EXISTENT_FILENAME, null);
+
+    Assert.assertEquals(
+        "Returns empty properties", Collections.emptySet(), testProperties.entrySet());
+  }
+
+  @Test
+  public void testAttemptLoadWithFileSystem() throws Exception {
+    Properties testProperties =
+        PropertiesLoader.attemptLoadWithFileSystem(propertiesFile.getPath(), null);
+
+    Assert.assertEquals(
+        "Returns the correct properties", exampleMap.entrySet(), testProperties.entrySet());
+  }
+
+  @Test
+  public void testAttemptLoadWithFileSystemFileDNE() throws Exception {
+    Properties testProperties =
+        PropertiesLoader.attemptLoadWithFileSystem(NON_EXISTENT_FILENAME, null);
+
+    Assert.assertEquals(
+        "Returns empty properties", Collections.emptySet(), testProperties.entrySet());
+  }
+
+  @Test
+  public void testAttemptLoadAsResource() throws Exception {
+    Properties testProperties =
+        PropertiesLoader.attemptLoadAsResource("/" + PROPERTIES_FILENAME, null);
```

Comment:
✏️  Use `String.format(...)` instead of concatenation.

Reply:
string format isn't necessary for a 2 item concat